ext/skeleton/config.m4.in

```diff
@@ -21,23 +21,31 @@ PHP_ARG_ENABLE([%EXTNAME%],
 if test "$PHP_%EXTNAMECAPS%" != "no"; then
   dnl Write more examples of tests here...
 
+  dnl Remove this code block if the library does not support pkg-config.
+  dnl PKG_CHECK_MODULES([LIBFOO], [foo])
+  dnl PHP_EVAL_INCLINE($LIBFOO_CFLAGS)
+  dnl PHP_EVAL_LIBLINE($LIBFOO_LIBS, %EXTNAMECAPS%_SHARED_LIBADD)
+
+  dnl Alternative library detection method using pkg-config.
```

Comment:
I think you can drop this block, unless there's any reason to use this over PKG_CHECK_MODULES? Might add an extra example above with `foo >= 1.2.3` to show how to do the version check.

Reply:
@nikic, I'm happy to add a version check example using PKG_CHECK_MODULES, but I kept the older method because @krakjoe said earlier, "do not remove the old (normal) way". Perhaps you two (and others) can discuss this and come to an agreement?

Reply:
The older method was meant as a check for using /usr/include check (without pkg-config), so the drop of the version check is ok because it is bloated (similar to what I was doing in the ffi version checking).

Reply:
So, this patch afteral still includes the "old" way - using /usr/include check. :)